pandas/tools/util.py

```diff
@@ -61,6 +61,42 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {'signed', 'unsigned'}, str, numpy.dtype, or
+               Python dtype, default None
+        If possible, downcast the resulting numerical data to the
+        specified numerical dtype in 'downcast'. If the size
+        of the data's dtype is smaller or equal to that of 'downcast',
+        then this parameter will be ignored.
+
+        If 'downcast' refers to a more generic dtype (e.g. int),
```

Comment:
Before I do, we need to resolve the issue <a href="https://github.com/pydata/pandas/pull/13425#discussion_r69729234">below</a>.